tests/migration_test.go

```diff
@@ -1551,6 +1551,9 @@ var _ = Describe("[Serial][rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][leve
 						Expect(err).ToNot(HaveOccurred())
 						return pdbs.Items
 					}, 3*time.Second, 500*time.Millisecond).Should(HaveLen(1))
+					By("waiting for VMI")
+					tests.WaitForSuccessfulVMIStartWithTimeout(vmi, 60)
```

Comment:
Probably not specific for this change, and I know many tests uses this, but it seems problematic to me to set hardcoded value, and different tests use different values.

Reply:
We could probably align it based on guest(booting time) and storage(provisioning time) but this is not a goal of this PR.